### PR TITLE
describe 2020-07-08 design meeting on the "path to lang-team membership"

### PR DESCRIPTION
**Pull request by nikomatsakis**
cc @rust-lang/lang -- does this seem like a good description?

### CODE REVIEW

File: posts/inside-rust/2020-07-08-lang-team-path-to-membership.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+[recording]: https://youtu.be/SeH-hZgDG1Y
+
+The premise of the meeting was that the lang team has never had a
+particularly clear *path to membership* -- i.e., it's been hard to say
+exactly what are the kinds of steps that one should be taking if you
+would like to become a member of the lang team. However, with the
+shift to [major change proposals] and in particular [project groups],
+we're starting to see what such a path looks like. Simultaneously,
+that same shift is helping to reveal that overall lang team bandwidth
+remains low, and that we could use to grow the team -- so thinking
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2020-07-08-lang-team-path-to-membership.md:24]: I'm having trouble understanding this sentence, perhaps changed mid-way through writing?

Comment (nikomatsakis) [reply posts/inside-rust/2020-07-08-lang-team-path-to-membership.md]: meh, I'm just going to cut the sentence.

File: posts/inside-rust/2020-07-08-lang-team-path-to-membership.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+Therefore, we discussed the idea of saying that liaisons don't have to
+be team members, they just have to be people who are heavily involved
+in the project and who can be trusted to create regular updates for
+the lang-team and keep the rest of the team in the loop. 
+
+In particular, this can also be a useful stepping stone towards full
+lang-team membership -- although it doesn't have to be. It's useful to
+have people serve as liaisons even if they don't really have time or
+interest in being on the lang team.
```
Comment (XAMPPRocky) [inline posts/inside-rust/2020-07-08-lang-team-path-to-membership.md:109]: I think there should be a conclusion on the liasions idea and the output of the meeting.

Comment (nikomatsakis) [reply posts/inside-rust/2020-07-08-lang-team-path-to-membership.md]: added

File: posts/inside-rust/2020-07-09-lang-team-path-to-membership.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Participate constructively in, and help facilitate, RFC discussion, issues, PRs, and other GitHub-based discussions
+    - Provide important technical points
+    - Help to drive discussions towards common understanding
+    - Understanding and documenting the positions and points being raised
+- Monitor and respond to communication in Zulip
+
+### A sketch for a path to membership
+
+The core idea for a path to membership is that we want some set of
+steps that let us see people doing the things we expect from lang-team
```
Comment (joshtriplett) [inline posts/inside-rust/2020-07-09-lang-team-path-to-membership.md:48]: Doing the things and demonstrating the qualities that we expect of lang team members

File: posts/inside-rust/2020-07-09-lang-team-path-to-membership.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+In particular, this can also be a useful stepping stone towards full
+lang-team membership -- although it doesn't have to be. It's useful to
+have people serve as liaisons even if they don't really have time or
+interest in being on the lang team.
+
+### Conclusion
+
+We concluded that we'll start experimenting with "non-team-member
+liaisons", and that people who are maybe interested in that role can
+reach out privately to Josh Triplett or myself. Further, we'll work to
```
Comment (joshtriplett) [inline posts/inside-rust/2020-07-09-lang-team-path-to-membership.md:115]: This post may make the rounds, and authorship information on the posts is a note at the top; you might want to give your name explicitly.

**Pull request merged**